shared/templates/csv/sshd_lineinfile.csv

```diff
@@ -1,3 +1,18 @@
 # <rule_id, parameter, value[, missing_parameter_pass]> (missing_parameter_pass: If set, the OVAL check will also pass if the parameter is not present in the configuration file. Values can be true or false. Default value is false.)
 # example of usage:
```

Comment:
You can remove the example of usage now as there will be multiple occurrences already that can be used as example